app/services/preserved_object_handler.rb

```diff
@@ -142,6 +133,27 @@ def update_version
 
   private
 
+  # runs the provided block.  if any of the expected exceptions are raised, returns a list
```

Comment:
I don't think we need these comments - can tell this by looking at the code and the caller/ where its used